docs/user/security/api-keys/index.asciidoc

```diff
@@ -23,15 +23,9 @@ image:images/api-keys.png["API Keys UI"]
 [[api-keys-service]]
 === {es} API key service
 
-The {es} API key service is automatically enabled when you configure
-{ref}/configuring-tls.html#tls-http[TLS on the HTTP interface].
-This ensures that clients are unable to send API keys in clear-text.
-
-When HTTPS connections are not enabled between {kib} and {es},
-you cannot create or manage API keys, and you get an error message.
-For more information, see the
-{ref}/security-api-create-api-key.html[{es} API key documentation],
-or contact your system administrator.
+The {es} API key service is automatically enabled. For instructions on disabling it, see {ref}/security-settings.html#api-key-service-settings[API key service settings].
```

Comment:
Basically copied this from ES docs with slight modification. But honestly, I'm wondering if we need this section in Kibana 8.0+ docs at all @jportner? I can see that it can be somewhat useful if API keys are disabled for some reason, and Kibana users try to figure this out. But suggesting that there is an option to disable API keys without explaining why anyone would want to do that, feels confusing to me.

Reply:
I agree, I think we can remove this section completely in 8.0+ docs.